### PR TITLE
Fix/toast notification hook (merge 3rd)

### PR DESCRIPTION
**Pull request by HaGuesto**
Probably there will be multiple conflicts and I will have to resolve merge conflict

### CODE REVIEW

**Comment by HaGuesto** [comment]
[2.0 Switch how notifications work (remove Apollo ReactiveVar)](https://trello.com/c/JdYekaxp/1043-20-switch-how-notifications-work-remove-apollo-reactivevar)

**Comment by HaGuesto** [comment]
I merged #542 into this PR so that you can merge one PR so you can merge it one other another

File: react/src/views/BoxCreate/BoxCreateView.tsx [hunk 1]
```diff
@@ -105,14 +106,14 @@ function BoxCreateView() {
   // Check the QR Code
   useEffect(() => {
     if (qrCodeExists.data?.qrExists === false) {
-      notificationVar({
+      createToast({
```
Comment (pylipp) [inline react/src/views/BoxCreate/BoxCreateView.tsx:109]: Why is `triggerError` not used here?

File: react/src/views/Box/BoxView.tsx [hunk 2]
```diff
@@ -218,18 +219,18 @@ function BTBox() {
...
   if (
+    loading ||
     updateBoxLocationMutationStatus.loading ||
     assignBoxToDistributionEventMutationStatus.loading ||
     unassignBoxFromDistributionEventMutationStatus.loading
   ) {
     return <APILoadingIndicator />;
   }
+
+  if (error) return <div />;
```
Comment (pylipp) [inline react/src/views/Box/BoxView.tsx:231]: Why not show an error toast here?

File: react/src/views/BoxCreate/BoxCreateView.tsx (hunk 1 above)
Comment (pylipp) [reply react/src/views/BoxCreate/BoxCreateView.tsx:109]: Clarified: we don't want the side effect of console error and logging to sentry, *only* the toast displayed

File: react/src/views/Box/BoxView.tsx (hunk 2 above)
Comment (pylipp) [reply react/src/views/Box/BoxView.tsx:231]: Clarified: error handling in BoxView and BoxEdit should be refactored (in the style of BoxCreate)

Comment (HaGuesto) [reply react/src/views/Box/BoxView.tsx:231]: done in #542

**Review by pylipp — APPROVED** [review]

**Pull request merged**